src/Titon/Context/Depository.hh

```diff
@@ -0,0 +1,357 @@
+<?hh // strict
+/**
+ * @copyright   2010-2015, The Titon Project
+ * @license     http://opensource.org/licenses/bsd-license.php
+ * @link        http://titon.io
+ */
+
+namespace Titon\Context;
+
+use Closure;
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionFunction;
+use ReflectionException;
+use Titon\Context\Definition\AbstractDefinition;
+use Titon\Context\Exception\AlreadyRegisteredException;
+
+/**
+ * The depository serves as a dependency injector. After registering an object,
+ * class, or callable with the depository, retrieving it will handle any necessary
+ * dependency injection and reflection resolution before returning the object for
+ * use.
+ *
+ * @package Titon\Context
+ */
+class Depository
+{
+    /**
+     * Hash of registered item definitions keyed by its alias or class name
+     *
+     * @var ItemContainer
+     */
+    protected ItemContainer $items = Map{};
```

Comment:
Lets add a space between all vector and map calls for consistency. `Map {}`, `Vector {}`, etc.